### PR TITLE
[Azure AD roles] [Admin units] Prerequisite roles

### PR DESCRIPTION
**Pull request by rolyon**

### CODE REVIEW

**Comment by rolyon** [comment]
@FaithOmbongi
When you have time, can you review this pull request?
thanks

File: api-reference/beta/api/administrativeunit-delete-members.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To remove a member from an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-delete-members.md:28]:
```suggestion
To remove a member from an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md [hunk 2]
```diff
@@ -25,6 +25,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | RoleManagement.ReadWrite.Directory |
 
+To remove a role assignment from an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-delete-scopedrolemembers.md:28]:
```suggestion
To remove a role assignment from an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/administrativeunit-delete.md [hunk 3]
```diff
@@ -25,6 +25,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To delete an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-delete.md:28]:
```suggestion
To delete an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/administrativeunit-post-members.md [hunk 4]
```diff
@@ -34,6 +34,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | Group.Create, Group.ReadWrite.All, Directory.ReadWrite.All |
 
+To add a member to an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-post-members.md:37]:
```suggestion
To add a member to an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/administrativeunit-post-scopedrolemembers.md [hunk 5]
```diff
@@ -25,6 +25,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | RoleManagement.ReadWrite.Directory |
 
+To assign Azure AD roles with an administrative unit scope, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-post-scopedrolemembers.md:28]:
```suggestion
To assign Azure AD roles with an administrative unit scope, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/administrativeunit-update.md [hunk 6]
```diff
@@ -24,6 +24,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To update an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/administrativeunit-update.md:27]:
```suggestion
To update an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/beta/api/directory-post-administrativeunits.md [hunk 7]
```diff
@@ -24,6 +24,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To create an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/beta/api/directory-post-administrativeunits.md:27]:
```suggestion
To create an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-delete-members.md [hunk 8]
```diff
@@ -23,6 +23,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To remove a member from an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-delete-members.md:26]:
```suggestion
To remove a member from an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md [hunk 9]
```diff
@@ -23,6 +23,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | RoleManagement.ReadWrite.Directory |
 
+To remove a role assignment from an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-delete-scopedrolemembers.md:26]:
```suggestion
To remove a role assignment from an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-delete.md [hunk 10]
```diff
@@ -23,6 +23,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To delete an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-delete.md:26]:
```suggestion
To delete an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-post-members.md [hunk 11]
```diff
@@ -25,6 +25,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To add a member to an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-post-members.md:28]:
```suggestion
To add a member to an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md [hunk 12]
```diff
@@ -23,6 +23,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | RoleManagement.ReadWrite.Directory |
 
+To assign Azure AD roles with an administrative unit scope, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-post-scopedrolemembers.md:26]:
```suggestion
To assign Azure AD roles with an administrative unit scope, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/administrativeunit-update.md [hunk 13]
```diff
@@ -22,6 +22,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To update an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/administrativeunit-update.md:25]:
```suggestion
To update an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

File: api-reference/v1.0/api/directory-post-administrativeunits.md [hunk 14]
```diff
@@ -22,6 +22,11 @@ One of the following permissions is required to call this API. To learn more, in
 |Delegated (personal Microsoft account) | Not supported.    |
 |Application | AdministrativeUnit.ReadWrite.All |
 
+To create an administrative unit, the calling principal must be assigned one of the following Azure AD roles:
```
Comment (FaithOmbongi) [inline api-reference/v1.0/api/directory-post-administrativeunits.md:25]:
```suggestion
To create an administrative unit, the calling principal must be assigned one of the following [Azure AD roles](/azure/active-directory/roles/permissions-reference):
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @rolyon for making these doc fixes. Approved with suggestions to link to the [Azure AD roles reference doc](docs.microsoft.com/azure/active-directory/roles/permissions-reference).

**Pull request merged**